strongbox-aql/src/test/java/org/carlspring/strongbox/aql/AqlParserTest.java

```diff
@@ -71,6 +71,84 @@ public void testLayoutSpecificKeywords()
         assertEquals(Pair.with(1, 115), errorPositionList.get(1));
     }
 
+    @Test
+    public void testValidDoubleQuotedQuerySpecificChars()
+    {
+        String query = "repository: \"releases#1\"";
+
+        AqlQueryParser aqlParser = new AqlQueryParser(query);
+        Map<Pair<Integer, Integer>, String> errorMap = null;
+
+        aqlParser.parseQuery();
+
+
+        logger.info(String.format("Query [%s] parse tree:\n[%s]", query, aqlParser));
+
+        assertFalse(aqlParser.hasErrors());
+
+    }
+
+
+    @Test
+    public void testValidSingleQuotedQuerySpecificChars()
+    {
+        String query = "repository: 'releases#1'";
```

Comment:
Repositories cannot have such names with special characters. Would you mind changing this to `releases` (here and everywhere else across the pull request)? Thanks! :)